prow/scripts/cluster-integration/kyma-gke-upgrade.sh

```diff
@@ -50,6 +50,11 @@ export KYMA_SCRIPTS_DIR="${KYMA_SOURCES_DIR}/installation/scripts"
 export TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS="${TEST_INFRA_SOURCES_DIR}/prow/scripts/cluster-integration/helpers"
 export KYMA_INSTALL_TIMEOUT="30m"
 export KYMA_UPDATE_TIMEOUT="15m"
+export UPGRADE_TEST_PATH="${KYMA_SOURCES_DIR}/tests/end-to-end/upgrade/chart/upgrade"
+export UPGRADE_TEST_HELM_TIMEOUT=10000
```

Comment:
add suffix what it really is, sec, milisec, minutes?
same comment for below variable